docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+| Frame Count                                      | frames_count                  | N/A                      |                 | INTEGER          |             --              |    0    | OPTIONAL                    | default=0                             | Number of frames associated to the tilt series.                                                                                                                                                                                                                                                                                                               |
+
+### 4. Tomogram Metadata
+
+## 4.1 Tomogram
+
+Describe the attributes of the tomograms.
+
+| Data elements                 | metadata key 1               | Sub-elements                | metadata key 2              | Value type                 | Unit  | Default | Requirement                                                           | Controlled vocabularies                                        | Description                                                                                                                                                                                                        |
+|-------------------------------|------------------------------|-----------------------------|-----------------------------|----------------------------|:-----:|:-------:|-----------------------------------------------------------------------|----------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Tomogram file name            | ???                          | N/A                         |                             | STRING                     |  --   |   --    | MUST                                                                  | =Tilt_experiment.name+Processing+Processing_software+\[no\]Ctf | Tomogram file name                                                                                                                                                                                                 | 
+| Tilt series file name         | ???                          | N/A                         |                             | reference                  |  --   |   --    | MUST                                                                  |                                                                | The name of the tilt series used to construct the tomograms                                                                                                                                                        |
```

Comment:
These exist in the google schema, but not in the implementation.

Reply:
This field seems useful - seems we should add it to our implementation?